### ISSUE TITLE
 No publishable resources for tag [] for publishing config file

### ISSUE THREAD

**Issue by chiva-song** [roach-php/core#129]
This happen for Laravel 9.19 when run command php artisan vendor:publish --provider='RoachPHP\Laravel\RoachServiceProvider' to publish config file.

**Comment by ksassnowski** [comment]
Did you install `roach-php/laravel`? My guess is that you might have only installed the `roach-php/core` package.

**Comment by chiva-song** [comment]
Yes, I did install before publishing a config file and I saw some dependencies was installed like roach-php/core package. Sent from Mail for Windows From: Kai SassnowskiSent: Thursday, June 1, 2023 1:37 PMTo: roach-php/coreCc: chiva-song; AuthorSubject: Re: [roach-php/core] No publishable resources for tag [] for publishing config file (Issue #129) Did you install roach-php/laravel? My guess is that you might have only installed the roach-php/core package.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***> Virus-free.www.avast.com
